external/soci/conanfile.py

```diff
@@ -68,7 +68,7 @@ def requirements(self):
         if self.options.with_mysql:
             self.requires("libmysqlclient/8.1.0")
         if self.options.with_postgresql:
-            self.requires("libpq/15.5")
```

Comment:
I wouldn't update dependency's dependencies in external. The recipes are written for a specific version by someone else so why bother. 

Update the dependency's dependency by updating the dependency.